README.md

```diff
@@ -2,31 +2,483 @@
 
-## Conclusion
+- The `Main` class contains a `main` method that creates an instance of `Motorcycle`
+  and implicitly calls the `toString` and explicitly calls the`rideThroughMud` methods on the object.
+  - We will evolve the `main` to call additional `Motorcycle` methods as we write them.
+    The `main` method will also instantiate a `Car` object and call its methods.
+- The `Motorcycle` class has fields for the `make` and `model`, along
+  with  fields `frontTire` and `rearTire` that reference instances of the `Tire` class.
+- The `Car` class has fields for the `make` and `model`, along
+  with an array named `tires` that stores references to 4 `Tire` objects.
+- The `Tire` class has instance variables `airPressure` and `clean`,
+  along with class constants `MIN_RECOMMENDED_PRESSURE` and `MAX_RECOMMENDED_PRESSURE`.
 
-A short one or two paragraph summary of the contents of the lessons, recapping
-the learning goals.
+```java
+public class Main {
+
+    public static void main(String[] args) {
+        Motorcycle myHarley = new Motorcycle("Harley Davidson", "Sportster");
+        System.out.println(myHarley);
+
+        System.out.println("Ride through mud");
+        myHarley.rideThroughMud();
+        System.out.println(myHarley);
+        
+    }
+}
+```
+
+## Visualizing Object Relations and Interactions
+
+
+```java
+public class Motorcycle {
+    
+    ...
+
+    public void rideThroughMud() {
+        frontTire.setClean(false);
+        rearTire.setClean(false);
+    }
+    
+    ...
+}
+```
+
+Let's consider the sequence of method calls that occur when the `rideThroughMud()` method
+is called by the `main` method.  This is depicted visually in the following **UML Sequence Diagram**.
+
+![sequence rideThroughMud](https://curriculum-content.s3.amazonaws.com/6676/java-multipleclasses/sequence_ridethroughmud.png)
+
+A sequence diagram displays the **lifeline** of
+each object as a vertical dashed line. A solid horizontal line
+represents a message from one object (or class) to another.
+In the diagram above, the messages represent  method calls
+`rideThroughMud()` and `setClean()`.
+
+Let's use the debugger to step through the method calls and
+observe how the object interactions affect object state.
+Set a breakpoint in the `main` method after the `Motorcycle`
+object has been created and the initial state printed
+at the line of code `myHarley.rideThroughMud();`.
+
+![breakpoint rideThroughMud](https://curriculum-content.s3.amazonaws.com/6676/java-multipleclasses/breakpoint_ridethroughmud.png)
+
+- Use 'Step Over' for the print statements.
+- Use 'Step Into' for the `Motorcycle` and `Tire` method calls.
+
+### Message #1 `myHarley.rideThroughMud()`
+
+Press the `debug` button to launch the debugger and stop at the breakpoint,
+then switch to the Java Visualizer view (the yellow arrow is added to show the
+current line of execution).  The `MotorCycle` object
```

Comment:
Lowercase c in `MotorCycle` to match the class name

Reply:
wow, good catch.